src/main/java/org/jenkinsci/plugins/nomad/NomadProvisioningStrategy.java

```diff
@@ -71,5 +83,40 @@
         LOGGER.log(Level.FINE, "Provisioning not complete, consulting remaining strategies");
         return NodeProvisioner.StrategyDecision.CONSULT_REMAINING_STRATEGIES;
     }
-}
 
+    private static void fireOnStarted(final Cloud cloud, final Label label,
+            final Collection<NodeProvisioner.PlannedNode> plannedNodes) {
+        for (CloudProvisioningListener cl : CloudProvisioningListener.all()) {
+            try {
+                cl.onStarted(cloud, label, plannedNodes);
+            } catch (Error e) {
+                throw e;
```

Comment:
This is useless, isn't it?

Reply:
Did copy from: https://github.com/jenkinsci/jenkins/blob/e27b310065b3c036b5fc9d123f1d1d99d3058c00/core/src/main/java/hudson/slaves/NodeProvisioner.java#L904-L917

Also there have been discussion about propagating that exception further in here: https://github.com/jenkinsci/jenkins/pull/2591/files#r84593689

Tell me what you think, from here it seems to me that this catch needs to stay

Reply:
Thanks for the link, looks like we can keep this here then :+1: